lib/src/Base/Algo/KarhunenLoeveSVDAlgorithm.cxx

```diff
@@ -237,15 +233,9 @@ void KarhunenLoeveSVDAlgorithm::run()
     values.setData(a);
     modesAsProcessSample.add(values);
     if (meshDimension == 1)
-    {
-      evaluation1D->setValues(values);
-      modes.add(*evaluation1D);
-    }
+      modes.add(PiecewiseLinearEvaluation(sample_.getMesh().getVertices().getImplementation()->getData(), values));
```

Comment:
You reverted my change

Reply:
Oups! Sorry, I messed a merge somewhere...

Reply:
Fixed.